src/pipelines.js

```diff
@@ -1,15 +1,15 @@
 /**
  * @file Pipelines provide a high-level, easy to use, API for running machine learning models.
- * 
+ *
```

Comment:
My editor trims the tailing spaces. Please let me know if you don't like this change. For review, you may set the option to "Hide whitespace".